src/gateway/kong-enterprise/dev-portal/enable.md

```diff
@@ -1,62 +1,91 @@
+    curl -i -X PATCH http://localhost:8001/workspaces/default \
+      --data "config.portal=true"
+    ```
+
+1. Access the Dev Portal for the workspace using the URL specified
+in the `KONG_PORTAL_GUI_HOST` variable:
+
+    ```sh
+    http://localhost:8003/default
+    ```
 
-   ```bash
-   curl -X PATCH http://localhost:8001/workspaces/default   --data "config.portal=true"
+{% endnavtab %}
+{% navtab Linux (kong.conf) %}
+
+1. To enable the Dev Portal, the following property must be set in the Kong
+configuration file ([`kong.conf`](/gateway/{{page.kong_version}}/production/kong-conf/)):
+
+   ```
+   portal = on
    ```
 
-   - This will expose the **default Dev Portal** at [http://localhost:8003/default](http://localhost:8003/default)
-   - The **Dev Portal Files endpoint** can be accessed at `:8001/files`
-   - The **Public Dev Portal Files API** can be accessed at `:8004/files`
+   {{site.base_gateway}} must be **restarted** for this value to take effect.
 
-## Enable Dev Portal with Docker installation
+   ```
+   kong reload
+   ```
 
-{:.note}
-> This feature is only available with a [{{site.konnect_product_name}} Enterprise](/gateway/{{page.kong_version}}/licenses) subscription.
+1. Enable Dev Portal for a workspace using one of the following methods:
 
-1. [Deploy a license](/gateway/{{page.kong_version}}/licenses/deploy).
+{% capture enable-portal %}
+{% navtabs %}
+{% navtab Kong Manager %}
 
-2. In your Docker container, set the Portal URL and set `KONG_PORTAL` to `on`:
+<!-- vale off -->
+1. Navigate to a workspace in Kong Manager.
+2. In the **Dev Portal** menu section, click **Overview**.
+3. Click the button to **Enable Developer Portal**.
+<!-- vale on -->
 
-    ```plaintext
-    echo "KONG_PORTAL_GUI_HOST=localhost:8003 KONG_PORTAL=on kong reload exit" \
-      | docker exec -i kong /bin/sh
-    ```
+{% endnavtab %}
+{% navtab Admin API %}
 
-    {:.note}
-    > The `HOSTNAME` for `KONG_PORTAL_GUI_HOST` should not be preceded by a protocol, for example, `http://`.
+```bash
+curl -i -X PATCH http://localhost:8001/workspaces/default \
+  --data "config.portal=true"
+```
 
-3. Execute the following command:
+{% endnavtab %}
+{% endnavtabs %}
+{% endcapture %}
 
-    <pre><code>curl -X PATCH --url http://<div contenteditable="true">{HOSTNAME}</div>:8001/workspaces/default \
-        --data "config.portal=true"</code></pre>
+{{ enable-portal | indent | replace: " </code>", "</code>"}}
 
-4. Access the Dev Portal for the default workspace using the URL specified
-in the `KONG_PORTAL_GUI_HOST` variable:
+{% endnavtab %}
+{% endnavtabs %}
 
-    <pre><code>http://<div contenteditable="true">{HOSTNAME}</div>:8003/default</code></pre>
+Enabling the Dev Portal exposes the following URLs:
+* The workspace's Dev Portal URL.
+For example, for the `default` workspace, the URL is: `http://localhost:8003/default`.
+* Dev Portal files endpoint: `http://localhost:8001/files`
+* Public Dev Portal files API: `http://localhost:8004/files`
```

Comment:
Maybe this could be moved to the opening paragraphs as a "here's what happens when you enable Dev Portal/why you'd want to enable the Dev Portal". Having it at the beginning might make it a little easier for customers to return to this page if they forget the URLs?